packages/pharos/src/components/tooltip/pharos-tooltip.ts

```diff
@@ -84,9 +84,7 @@ export class PharosTooltip extends OverlayElement {
   );
 
   protected get content(): HTMLElement | Text {
-    return Array.prototype.slice
-      .call(this._contentNodes)
-      ?.find((node) => node.textContent && node.nodeName === '#text');
+    return Array.prototype.slice.call(this._contentNodes)?.find((node) => node.textContent);
```

Comment:
```suggestion
    return Array.prototype.slice.call(this._contentNodes)?.find((node) => node.textContent && node.nodeName !== '#comment');
```
To ensure we don't pick up one of Lit's comment nodes